### PR TITLE
Extract new strings (2023-10-24)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -3523,6 +3543,10 @@ Private Tab</source>
         <source>Make %@ your go-to browser</source>
         <note>String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Welcome.Title.TreatementA.v120" xml:space="preserve">
+        <source>We love keeping you safe</source>
+        <note>String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3548]: Nit for comment: there's no placeholder in the string

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+        <source>Show Less</source>
+        <note>Title of the button that shows less reviews in the review highlights displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.MoreButton.Title.v120" xml:space="preserve">
+        <source>Show More</source>
+        <note>Title of the button that shows more reviews in the review highlights displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.Packaging.Title.v120" xml:space="preserve">
+        <source>Packaging</source>
+        <note>Section title of the review highlights displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3789]: I'm not sure what "Packaging" actually refers to here. It seems the comment is too generic, please detail a bit more as to what "packaging" really entails

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+        <source>This could take about 60 seconds.</source>
+        <note>Description for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.InfoCard.ProgressAnalysis.Title.v120" xml:space="preserve">
+        <source>Checking review quality</source>
+        <note>Title for info card when the product is in analysis mode</note>
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.AnalyzerButton.Title.v120" xml:space="preserve">
+        <source>Check Review Quality</source>
+        <note>Text for the analyzer button, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3849]: The comment might be a bit confusing to localizers and seems more dev oriented, from this part "from No Analysis Card View displayed in the shopping review quality bottom sheet". 
Something more like "text for the analyzer button, when the analysis can be updated for a product" (or something along those lines, if there's a better description)

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.BodyLabel.Title.v120" xml:space="preserve">
+        <source>To know whether this product’s reviews are reliable, check the review quality. It only takes about 60 seconds.</source>
+        <note>Text for the body label, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3853]: "from No Analysis Card View displayed in the shopping review quality bottom sheet"
-> also a bit confusing for localizers. 
Could be just "text for body label, to check reliability of product"

File: templates/firefox-ios.xliff [hunk 5]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.HeadlineLabel.Title.v120" xml:space="preserve">
+        <source>No info about these reviews yet</source>
+        <note>Text for the Headline Label, from No Analysis Card View displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3857]: same comment about being dev oriented ("bottom sheet" may be hard to understand)

File: templates/firefox-ios.xliff [hunk 6]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      </trans-unit>
+      <trans-unit id="Shopping.NoAnalysisCard.InProgress.Body.Label.v120" xml:space="preserve">
+        <source>This could take about 60 seconds.</source>
+        <note>Description of the No Analysis Card when the analysis is in progress displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3861]: same comment about being mostly a dev oriented comment

File: templates/firefox-ios.xliff [hunk 7]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.FirstParagraph.Title.v120" xml:space="preserve">
+        <source>See how reliable product reviews are on %1$@ before you buy. Review checker, an experimental feature from %2$@, is built right into the browser. It works on %3$@ and  %4$@, too.
+
+Using the power of %5$@ by %6$@, we help you avoid biased and inauthentic reviews. Our AI model is always improving to protect you as you shop.</source>
+        <note>Label for the first paragraph of the Shopping Experience Opt In onboarding Card (Fakespot). The first parameter will be the website the user is coming from when viewing this screen (default Amazon). The second parameter will be replaced by the app name. The third and fourth parameters will be the other two websites that are currently supported (Amazon, Best Buy or Walmart) besides the one used for the first parameter. The fifth and sixth parameter are the Fakespot app name and the company name of Mozilla.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3875]: Having so many parameters in one string has a good chance of creating confusion for localizers, resulting in awkward/wrong translations.
For browser and android, these are two [strings](https://transvision.mozfr.org/?recherche=See+how+reliable+product+reviews&repo=all_projects&sourcelocale=en-US&locale=fr&search_type=strings) (stopping at the 4th parameter).
I'd suggest to mirror that here too

File: templates/firefox-ios.xliff [hunk 8]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+        <source>Not now</source>
+        <note>Text for the secondary button of the Shopping Experience Opt In onboarding Card (Fakespot)</note>
+      </trans-unit>
+      <trans-unit id="Shopping.OptInCard.TermsOfUse.Button.Title.v120" xml:space="preserve">
+        <source>Terms of use</source>
+        <note>Show Firefox Browser Terms of Use page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReliabilityScore.Grade.A11y.Label.v120" xml:space="preserve">
+        <source>Grade %@</source>
+        <note>Accessibility label for the Grade labels used in 'How we determine review quality' card and 'How reliable are these reviews' card displayed in the shopping review quality bottom sheet. The placeholder will be replaced by a grade letter (e.g. A).</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3903]: I'd just add that it's a grading system from A-F

File: templates/firefox-ios.xliff [hunk 9]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      <trans-unit id="Shopping.ReviewQuality.ReliabilityRating.DF.Description.v120" xml:space="preserve">
+        <source>Unreliable reviews</source>
+        <note>Description of the reliability ratings for rating 'D' and 'F' displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.AdjustedRating.Label.v120" xml:space="preserve">
+        <source>The *adjusted rating* is based only on reviews we believe to be reliable.</source>
+        <note>Adjusted rating label from How we determine review quality card displayed in the shopping review quality bottom sheet. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.Collapsed.AccessibilityLabel.v120" xml:space="preserve">
+        <source>How we determine review quality Card Collapsed</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3926]: This sounds a bit confusing when finishing with "Card Collapsed"

File: templates/firefox-ios.xliff [hunk 10]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+        <note>Accessibility label for the up chevron, from 'How we determine review quality' card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.Expanded.AccessibilityLabel.v120" xml:space="preserve">
+        <source>How we determine review quality Card Expanded</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3930]: same comment as above

File: templates/firefox-ios.xliff [hunk 11]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+        <source>We assign each product’s reviews a *letter grade* from A to F.</source>
+        <note>Label of the sub headline from How we determine review quality card displayed in the shopping review quality bottom sheet. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.UnreliableReviews.Label.v120" xml:space="preserve">
+        <source>Unreliable reviews. We believe the reviews are likely fake or from biased reviewers.</source>
+        <note>Unreliable reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Collapsed.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Collapsed</source>
+        <note>Accessibility label for the up chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3967]: I'm not sure I understand this action

File: templates/firefox-ios.xliff [hunk 12]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Expanded.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Expanded</source>
+        <note>Accessibility label for the down chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3971]: same as above, I'm not sure this will be clear to localizers

File: templates/firefox-ios.xliff [hunk 13]
```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
...
+      <trans-unit id="Shopping.SettingsCard.Label.Title.v120" xml:space="preserve">
+        <source>Settings</source>
+        <note>Title of the settings card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.RecommendedProducts.Label.v120" xml:space="preserve">
+        <source>Show products recommended by %@</source>
+        <note>Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.TurnOff.Button.v120" xml:space="preserve">
+        <source>Turn Off Review Checker</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3986]: want to double check that title case is required here (I think it may?)

**Review by Delphine — CHANGES_REQUESTED** [review]
An additional comment i have and that I didn't keep adding is that the PR seems to get very developer comment oriented vs l10n. So for example, there are comments that all refer to "bottom sheet", and I think we can drop those as those strings are quite explicit already, and can just call out more where the string will appear in context.
Flagging @jnrahme - haven't checked who individually landed those strings since there are a lot. thanks!

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (jnrahme) [reply templates/firefox-ios.xliff:3548]: resolved

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (jnrahme) [reply templates/firefox-ios.xliff:3789]: Resolved

File: templates/firefox-ios.xliff [hunk 14]
```diff
@@ -3507,8 +3523,12 @@ Private Tab</source>
         <source>%@ puts people over profits and defends your privacy as you browse.</source>
         <note>String used to describes the description of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
       </trans-unit>
-      <trans-unit id="Onboarding.Welcome.Description.v114" xml:space="preserve">
-        <source>%@ puts people over profits and defends your privacy as you browse.</source>
+      <trans-unit id="Onboarding.Welcome.Description.TreatementA.v120" xml:space="preserve">
+        <source>Our non-profit backed browser helps stop companies from secretly following you around the web.</source>
+        <note>String used to describes the description of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
```
[the remark this answers is not in the document]
Comment (jnrahme) [reply templates/firefox-ios.xliff:3528]: Resolved

File: templates/firefox-ios.xliff (hunk 3 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3849]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

File: templates/firefox-ios.xliff (hunk 4 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3853]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

File: templates/firefox-ios.xliff (hunk 9 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3926]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

File: templates/firefox-ios.xliff (hunk 10 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3930]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

File: templates/firefox-ios.xliff (hunk 12 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3971]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

File: templates/firefox-ios.xliff (hunk 13 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff:3986]: Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940

**Comment by jnrahme** [comment]
https://github.com/mozilla-mobile/firefox-ios/pull/16963

File: templates/firefox-ios.xliff (hunk 5 above)
Comment (thatswinnie) [reply templates/firefox-ios.xliff:3857]: Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16964

File: templates/firefox-ios.xliff (hunk 6 above)
Comment (thatswinnie) [reply templates/firefox-ios.xliff:3861]: Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16964

File: templates/firefox-ios.xliff (hunk 8 above)
Comment (thatswinnie) [reply templates/firefox-ios.xliff:3903]: Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16964

**Review by Delphine — CHANGES_REQUESTED** [review]
@thatswinnie Just spotted and commented on one nit, and then I will be able to expose the strings on Pontoon. Thanks everyone for the updates!

File: templates/firefox-ios.xliff (hunk 7 above)
Comment (razvanlitianu) [reply templates/firefox-ios.xliff]: Will be fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16979

Comment (razvanlitianu) [reply templates/firefox-ios.xliff]: PR will land for v121

**Comment by thatswinnie** [comment]
@Delphine before this gets merged we need to merge [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/16990) and update the strings here.

**Review by Delphine — APPROVED** [review]

**Pull request merged**